app/controllers/v0/sessions_controller.rb

```diff
@@ -33,6 +33,9 @@ def saml_callback
       if @saml_response.is_valid? && persist_session_and_user
         async_create_evss_account(@current_user)
         redirect_to SAML_CONFIG['relay'] + '?token=' + @session.token
+
+        obscure_token = Session.obscure_token(@session.token)
+        Rails.logger.info("Logged in user with id #{@session.uuid}, token #{obscure_token}")
```

Comment:
In order to do the audits described in the referenced issue, we have to have a way to tie the requests back to the user. The session's UUID is logged as the connection to the user, and this line connects _that_ to an obscured version of the token so we can get all of the log events associated with that user. Still, the :uuid log tag is needed in order to group events in the log where the user is unauthorized so that it's possible to see the full logs from a single request.